x-pack/plugins/security_solution/server/endpoint/routes/artifacts/download_exception_list.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IRouter } from 'src/core/server';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { authenticateAgentWithAccessToken } from '../../../../../ingest_manager/server/services/agents/authenticate';
```

Comment:
Can probably access this through the start interface, so we can get rid of this import.